packages/@aws-cdk/aws-route53-patterns/README.md

```diff
@@ -9,12 +9,34 @@
 ---
 <!--END STABILITY BANNER-->
 
-This library contains commonly used patterns for Route53.
-
+This library provides higher-level Amazon Route 53 constructs which follow common
+architectural patterns.
 
 ## HTTPS Redirect
 
-This construct allows creating a simple domainA -> domainB redirect using CloudFront and S3. You can specify multiple domains to be redirected.
+If you want to speed up delivery of your web content, you can use Amazon CloudFront,
+the AWS content delivery network (CDN). CloudFront can deliver your entire website
+including dynamic, static, streaming, and interactive content—by using a global
+network of edge locations. Requests for your content are automatically routed to the
+edge location that gives your users the lowest latency.
+
+This construct allows creating a redirect from domainA to domainB using Amazon
+CloudFront and Amazon S3. You can specify multiple domains to be redirected.
+[Learn more](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-to-cloudfront-distribution.html) about routing traffic to a CloudFront web distribution.
+
+The `HttpsRedirect` constructs creates:
+
+* Amazon CloudFront distribution - makes website available from data centres
+  around the world
+* Amazon S3 bucket - used to store content that CloudFront will use as the origin
```

Comment:
This construct creates a bucket with a redirect configuration (`websiteRedirect`) that will remain empty. It is only used for the redirection.

(taht's why it uses `RemovalPolicy.DESTROY`)